addons/auth/config/ember-try.js

```diff
@@ -7,18 +7,18 @@ module.exports = async function () {
   return {
     scenarios: [
       {
-        name: 'ember-lts-3.24',
+        name: 'ember-lts-3.28',
         npm: {
           devDependencies: {
-            'ember-source': '~3.24.3',
+            'ember-source': '~3.28.0',
           },
         },
       },
       {
-        name: 'ember-lts-3.28',
+        name: 'ember-lts-4.4',
```

Comment:
I was planning a first iteration with Ember 4.0 to validate all the deps and later on jump to 4.4 but I guess we can jump directly to 4.4